README.md

```diff
@@ -116,6 +117,18 @@ ecs:
     - authentication
     type:
     - info
+  user:
+    name: vagrant
+    id: "1000"
+    effective:
+      name: root
+      id: "0"
+    target: {}
+    changes: {}
+  group:
+    effective: {}
```

Comment:
I think the only one having `effective` etc is the `user` field https://www.elastic.co/guide/en/ecs/master/ecs-user-usage.html.

For the effective group would be something like `user.effective.group.*`

Reply:
Good catch. `group.effective` was unused, but `group.target` is used for group creation/deletion/changes.

I think the best for now is to set `group.id` in those cases? Because it's mapping actions like User A creates Group B. Another option is `user.target.group`? But that doesn't make a lot of sense without filling the user fields in `user.target`.

/cc @andrewstucki

Reply:
I think as far as `user` is modifying/creating a group does does not belong to a different user, is probably fine having both user and group at the root. Maybe @ebeahan can bring some guideline

Reply:
Yes, I agree with @marc-gr that using the top-level `user.*` and `group.*` is most appropriate in this case.